shared/menubar/files.desktop.js

```diff
@@ -36,6 +36,13 @@ type FilesPreviewProps = {|
   userTlfUpdates: Array<UserTlfUpdateRowProps>,
 |}
 
+// Force word wrapping in long filenames so widget does not expand horizontally.
+const wordWrapFilename = Styles.platformStyles({
+  isElectron: {
+    wordBreak: 'break-word',
+  },
+})
```

Comment:
Can you put this in `styles` at the bottom?

Reply:
yes! sorry I didn't notice there is a style dict